src/BitStringAddresses/fockaddress.jl

```diff
@@ -644,13 +684,24 @@ Base.show(io::IO, ::MIME"text/plain", i::FermiFSIndex) = show(io, i)
 
 Iterator over occupied modes in address. `N` is the number of fermions. See [`occupied_modes`](@ref).
 """
-struct FermiOccupiedModes{N,S}<:OccupiedModeIterator
+struct FermiOccupiedModes{N,S} <: ModeIterator
     storage::S
 end
 
 Base.length(::FermiOccupiedModes{N}) where {N} = N
 Base.eltype(::FermiOccupiedModes) = FermiFSIndex
 
+"""
+    FermiUnoccupiedModes{N,S<:BitString}
```

Comment:
```suggestion
    FermiUnoccupiedModes{N}
```
No need to specify the storage type here. The docstring is for the more general type.